runnc-cont/config.go

```diff
@@ -0,0 +1,37 @@
+package runnc_cont
+
+// Config configuration to create a runnc-cont
+type Config struct {
+	// NablaRunBin is the path to 'nabla-run' binary.
+	NablaRunBin string
+
+	NablaRunArgs []string
+
+	// UniKernelBin is the path to 'unikernel' binary.
+	UniKernelBin string
+
+	// Tap tap device. (e.g. tap100)
+	Tap string
+
+	IPAddress string
+	IPMask    int
+	GateWay   string
+
+	// Memory max memory size in MBs.
+	Memory int64
+
+	// IsInDocker means running in a Docker container or not.
+	IsInDocker bool
+
+	// IsInKubernetes means running in a Kubernetes cluster or not.
+	IsInKubernetes bool
+
+	// Disk is the path to disk
+	Disk string
```

Comment:
I think we want to be able to support multiple disks eventually - and mount them on different mount points. So perhaps a `Volumes []string` would be more appropriate here (and for now we just do it on the first volume) - On that note, let's just add a comment saying currently, only one (first) volume is supported. What do you think?